app/services/HCAService.js

```diff
@@ -12,6 +12,8 @@ import { GetStoreData, SetStoreData } from '../helpers/General';
 import languages from '../locales/languages';
 import LocationService from './LocationService';
 
+const { AUTHORITIES_LIST_URL } = env;
+
 /**
  * Singleton class to interact with health care authority data
```

Comment:
I don't believe this class is actually used anymore. (The methods are used in the permissions context, but I believe everything ends as a no-op)

Reply:
Okay - great to know. I will see if removing it makes sense in this PR, if not, we can remove it in a follow up PR.

Reply:
great, we should remove the dead code then.

Reply:
I will do this on a new branch/PR